tutorials/s4sdk-caching/s4sdk-caching.md

```diff
@@ -83,14 +83,14 @@ final ResilienceConfiguration.CacheConfiguration cacheConfig =
 myResilienceConfig.cacheConfiguration(cacheConfig);                         
 ```
 
-As mentioned above, we use our `ResilienceConfiguration` to integrate the caching functionality, which is described in a `CacheConfiguration`. There are two steps to our configuration:
+As mentioned above, the `ResilienceConfiguration` is used to integrate the caching functionality, which is described in a `CacheConfiguration`. There are two steps to your configuration:
 
 1. Determine how long objects are to be cached
 2. Declare the parameters that need to be stored together with the cached data
 
-The first step is obviously necessary, since we want to store the data for a limited amount of time. The longer we keep the cached information, the more outdated it will become. How long you want to keep data cached depends on your specific use case. How fast do you expect the information to be outdated? How frequently will the data be accessed? The timeout sets a trade-off between the data being up to date and the application being responsive.
+The first step is obviously necessary, since data should be stored for a limited time only. The longer you keep the cached information, the more outdated it will become. How long you want to keep data cached depends on your specific use case. How fast do you expect the information to be outdated? How frequently will the data be accessed? The timeout sets a trade-off between the data being up to date and the application being responsive.
```

Comment:
```suggestion
The first step is obviously necessary since data should be stored for a limited time only. The longer you keep the cached information, the more outdated it will become. How long you want to keep data cached depends on your specific use case. How fast do you expect the information to be outdated? How frequently will the data be accessed? The timeout sets a trade-off between the data is up to date and the application is responsive.
```